Dockerfile

```diff
@@ -13,9 +13,7 @@ RUN mvn net.revelc.code.formatter:formatter-maven-plugin:format install -DskipTe
 #
 # RUN THE APPLICATION
 #
-FROM openjdk:18-ea-bullseye
-RUN apt-get update & apt-get upgrade
-
+FROM amazoncorretto:17-al2023-jdk
```

Comment:
We are now going from 18 to 17. Perhaps we could go up instead? Locally I run corretto 19.0.2 and this works fine with the server. That would be image 19.0.2-alpine3.17